### ISSUE TITLE
Temporary library and Indipendet ASP

### ISSUE THREAD

**Issue by pabloto** [halcyon-tech/vscode-ibmi#465]
Hi, 

every time I open Code for IBM i extension I get this warning:

![image](https://user-images.githubusercontent.com/9719313/148510589-fdff751c-0445-4d9f-8c79-55ab807744bd.png)

I create a library named vscode on our IASP, Do I have to put it into SYSBAS?

Many thanks

**Comment by worksofliam** [comment]
You should check the Code for IBM i output channel after you have connected and paste that output here. Specifically you should look for the `CRTLIB`.

For example, here the output for mines

```
/home/alan3/deleteme/: system "CRTLIB LIB(ILEDITOR) TEXT('Code for i temporary objects. May be cleared.')"
{
    "code": 255,
    "signal": null,
    "stdout": "",
    "stderr": "CPF2111: Library ILEDITOR already exists."
}
```

**Comment by pabloto** [comment]
Hi Liam,

here's all Code for IBM i output:

.: pwd
{
    "code": null,
    "signal": null,
    "stdout": "/DIRECTA1/home/paolos",
    "stderr": ""
}

/DIRECTA1/home/paolos: pwd
{
    "code": null,
    "signal": null,
    "stdout": "/DIRECTA1/home/paolos",
    "stderr": ""
}

/DIRECTA1/home/paolos: echo "liblist" | /QOpenSys/usr/bin/qsh
{
    "code": null,
    "signal": null,
    "stdout": "USYS        SYS\nQSYS        SYS\nQSYS2       SYS\nQHLPSYS     SYS\nQUSRSYS     SYS\nQSHELL      PRD\nPAOLOS      CUR\nQGPL        USR\nQTEMP       USR",
    "stderr": ""
}

/DIRECTA1/home/paolos: system "CRTLIB LIB(VSCODE) TEXT('Code for i temporary objects. May be cleared.')"
{
    "code": 255,
    "signal": null,
    "stdout": "",
    "stderr": "CPF2158: La libreria VSCODE esiste nell'unit� ASP DIRECTA1.\nCPF2111: La libreria VSCODE esiste gi�."
}

/DIRECTA1/home/paolos: ls -p /QOpenSys/pkgs/bin/
{
    "code": null,
    "signal": null,
    "stdout": "2to3\n2to3-3.6\n2to3-3.9\n5250keys\n7z\n7za\n7zr\naclocal\naclocal-1.15\nacountry\nactivemq\nactivemq-admin\naddftinfo\nadig\nadjust-xcoff-header\nafmtodit\nahost\nalternatives\nanacron\nansible\nansible-config\nansible-connection\nansible-console\nansible-doc\nansible-galaxy\nansible-inventory\nansible-playbook\nansible-pull\nansible-test\nansible-vault\nant\nantRun\napplydeltaiso\napplydeltarpm\napropos\naria_chk\naria_dump_log\naria_ftdump\naria_pack\naria_read_log\nasn1Coding\nasn1Decoding\nasn1Parser\natob\nautoconf\nautoexpect\nautoheader\nautom4te\nautomake\nautomake-1.15\nautopoint\nautoreconf\nautoscan\nautossh\nautoupdate\nawk\nbase32\nbase64\nbasename\nbash\nbashbug\nbison\nbrotli\nbsdcat\nbsdcpio\nbsdtar\nbsqldb\nbsqlodbc\nbtoa\nbunzip2\nbzcat\nbzcmp\nbzdiff\nbzegrep\nbzfgrep\nbzgrep\nbzip2\nbzip2recover\nbzless\nbzmore\nc2ph\ncal\ncaptoinfo\ncat\ncatman\ncc\nccache\nccmake\ncertutil\nchcon\nchem\nchgrp\nchmod\nchown\nchroot\nchroot_setup\nchsh\ncjpeg\ncksum\nclear\nclusterdb\ncmake\ncmp\ncmsutil\ncol\ncolcrt\ncolrm\ncolumn\ncombinedeltarpm\ncomm\ncorelist\ncp\ncpack\ncpan\ncpio\ncpp\ncpp-6\ncpp-6.bin\ncreatedb\ncreaterepo\ncreateuser\ncrlutil\ncrond\ncronnext\ncrontab\ncsplit\nctest\ncups-config\ncurl\ncurl-config\ncurve_keygen\ncut\ncwebp\ncxpm\nc_rehash\nc++\ndatacopy\ndate\ndb2util\ndb_archive\ndb_checkpoint\ndb_deadlock\ndb_dump\ndb_hotbackup\ndb_load\ndb_printlog\ndb_recover\ndb_sql\ndb_stat\ndb_upgrade\ndb_verify\ndd\ndebuginfo-install\ndefncopy\nderb\nderdump\ndf\ndiff\ndiff3\ndir\ndircolors\ndirmngr\ndirmngr-client\ndirname\ndislocate\ndjpeg\ndltest\ndos2unix\ndropdb\ndropuser\ndrpmsync\ndu\ndumpsexp\ndvipdf\ndwebp\neasy_install-2.7\necho\necpg\negrep\nenc2xs\nencguess\nenv\nenvsubst\neps2eps\neqn\neqn2graph\nescapesrc\nevent_rpcgen.py\nex\nexpand\nexpect\nexpr\nf2py3.6\nf95\nfactor\nfalse\nfax2ps\nfax2tiff\nfc-cache\nfc-cat\nfc-conflist\nfc-list\nfc-match\nfc-pattern\nfc-query\nfc-scan\nfc-validate\nfgrep\nfile\nfind\nfind-repos-of-install\nfisql\nfix-rpath\nflex\nflex++\nfmt\nfold\nfragiso\nfreebcp\nfunzip\ngapplication\ngawk\ngcc\ngcc-6\ngcc-6.bin\ngcc-ar\ngcc-ar-6\ngcc-ar-6.bin\ngcc-nm\ngcc-nm-6\ngcc-nm-6.bin\ngcc-ranlib\ngcc-ranlib-6\ngcc-ranlib-6.bin\ngcov\ngcov-6\ngdb\ngdbmtool\ngdbm_dump\ngdbm_load\ngdbus\ngdbus-codegen\ngdiffmk\ngenbrk\ngenccode\ngencfu\ngencmn\ngencnval\ngendict\ngendiff\ngennorm2\ngenrb\ngensprep\ngetopt\ngettext\ngettextize\ngettext.sh\ngfortran\ngfortran-6\ngfortran-6.bin\ngif2rgb\ngifbuild\ngifclrmp\ngifecho\ngiffix\ngifinto\ngiftext\ngiftool\ngio\ngio-querymodules\ngit\ngit-cvsserver\ngit-receive-pack\ngit-shell\ngit-upload-archive\ngit-upload-pack\nglib-compile-resources\nglib-compile-schemas\nglib-genmarshal\nglib-gettextize\nglib-mkenums\nglilypond\ngmake\ngobject-query\ngperf\ngperl\ngpg\ngpgconf\ngpgparsemail\ngpgrt-config\ngpgscm\ngpgsm\ngpgtar\ngpgv\ngpg-agent\ngpg-connect-agent\ngpg-error\ngpg-error-config\ngpg-wks-server\ngpinyin\ngrap2graph\ngrep\ngresource\ngrn\ngrodvi\ngroff\ngroffer\ngrog\ngrolbp\ngrolj4\ngropdf\ngrops\ngrotty\ngroups\ngs\ngsbj\ngsdj\ngsdj500\ngsed\ngsettings\ngslj\ngslp\ngsnd\ngtester\ngtester-report\ngunzip\ngzexe\ngzip\ng++\ng++-6\ng++-6.bin\nh2ph\nh2xs\nhardlink\nhb-ot-shape-closure\nhb-shape\nhb-subset\nhb-view\nhead\nhelp2man\nhexdump\nhmac256\nhostid\nhpftodit\niconv\nicuinfo\nicupkg\nicu-config\nid\nidle3\nidle3.6\nidle3.9\nidn2\nifnames\nigawk\nindxbib\ninfo\ninfocmp\ninfotocap\ninitdb\ninnochecksum\ninstall\ninstall-info\ninstmodsh\nipcmk\nisosize\nisql\niusql\njmacs\njoe\njoin\njpegtran\njpico\njq\njson_pp\njstar\nkbxutil\nkill\nksba-config\nless\nlessecho\nlesskey\nlexgrog\nlftp\nlftpget\nlibassuan-config\nlibgcrypt-config\nlibnetcfg\nlibpng16-config\nlibpng-config\nlibtool\nlibtoolize\nlink\nlistsuites\nlkbib\nln\nlocate\nlogname\nlook\nlookbib\nlp5250d\nlprsetup.sh\nls\nlua\nluac\nlzcat\nlzcmp\nlzdiff\nlzegrep\nlzfgrep\nlzgrep\nlzip\nlzless\nlzma\nlzmadec\nlzmainfo\nlzmore\nm4\nmac2unix\nmake\nmakeconv\nmakedeltaiso\nmakedeltarpm\nmakeinfo\nman\nmandb\nmanpath\nman-recode\nmc\nmcdiff\nmcedit\nmcookie\nmcview\nmd5sum\nmergerepo\nmesg\nminisign\nmkdir\nmkfifo\nmknod\nmktemp\nmmroff\nmodifyrepo\nmodutil\nmpicalc\nmsgattrib\nmsgcat\nmsgcmp\nmsgcomm\nmsgconv\nmsgen\nmsgexec\nmsgfilter\nmsgfmt\nmsggrep\nmsginit\nmsgmerge\nmsgunfmt\nmsguniq\nmsql2mysql\nmv\nmvn\nmvnDebug\nmvnyjp\nmyisamchk\nmyisamlog\nmyisampack\nmyisam_ftdump\nmysql\nmysqlaccess\nmysqladmin\nmysqlbinlog\nmysqlcheck\nmysqld\nmysqldump\nmysqldumpslow\nmysqld_multi\nmysqld_safe\nmysqld_safe_helper\nmysqlhotcopy\nmysqlimport\nmysqlshow\nmysqlslap\nmysql_convert_table_format\nmysql_find_rows\nmysql_fix_extensions\nmysql_install_db\nmysql_plugin\nmysql_secure_installation\nmysql_setpermission\nmysql_tzinfo_to_sql\nmysql_upgrade\nmysql_waitpid\nmytop\nmy_print_defaults\nnamei\nnano\nncdu\nncurses6-config\nncursesw6-config\nneeds-restarting\nneqn\nngettext\nnginx\nnice\nninja\nnl\nnode\nnodever\nnohup\nnologin\nnpm\nnproc\nnpth-config\nnpx\nnroff\nnspr-config\nnss-config\nnss-policy-check\nnumfmt\nocspclnt\nod\nodbcinst\nodbc_config\noid2name\nonig-config\nopenssl\nosql\np11-kit\npackage-cleanup\npal2rgb\npango-view\npaste\npatch\npathchk\npcre2grep\npcre2test\npcre2-config\npcregrep\npcretest\npcre-config\npdf2dsc\npdf2ps\npdfmom\npdfroff\npdftexi2dvi\nperl\nperl5.24.1\nperlbug\nperldoc\nperlivp\nperlthanks\nperror\npf2afm\npfbtopfa\npfbtops\npgbench\npg_archivecleanup\npg_basebackup\npg_checksums\npg_config\npg_controldata\npg_ctl\npg_dump\npg_dumpall\npg_isready\npg_receivewal\npg_recvlogical\npg_resetwal\npg_restore\npg_standby\npg_test_fsync\npg_test_timing\npg_upgrade\npg_waldump\npic\npic2graph\npiconv\npigz\npinentry\npinentry-curses\npinky\npip2\npip2.7\npip3\npip3.6\npip3.9\npk12util\npkgdata\npkg-config\npl2pm\npngfix\npng-fix-itxt\npod2html\npod2man\npod2texi\npod2text\npod2usage\npodchecker\npodselect\npostgres\npostmaster\npost-grohtml\npowerpc-ibm-os400-pkg-config\npp\npphs\nppm2tiff\npr\npreconv\npre-grohtml\nprintafm\nprintenv\nprintf\nprotoc\nprove\nps2ascii\nps2epsi\nps2pdf\nps2pdf12\nps2pdf13\nps2pdf14\nps2pdfwr\nps2ps\nps2ps2\npsql\npstruct\nptar\nptardiff\nptargrep\nptx\npv\npwd\npydoc\npydoc3\npydoc3.6\npydoc3.9\npython\npython2\npython2.7\npython2.7-config\npython2-config\npython3\npython3.6\npython3.6m\npython3.6m-config\npython3.6-config\npython3.9\npython3.9-config\npython3-config\npyvenv-3.6\nR\nraw2tiff\nrdjpgcom\nreadlink\nrealpath\nrecode-sr-latin\nredis-benchmark\nredis-check-aof\nredis-check-rdb\nredis-cli\nredis-sentinel\nredis-server\nrefer\nreindexdb\nrename\nrenice\nreplace\nrepoclosure\nrepodiff\nrepomanage\nrepoquery\nreposync\nrepotrack\nrepo-graph\nrepo-rss\nreset\nresolveip\nresolve_stack_dump\nrev\nrftp\nrjoe\nrm\nrmdir\nrnano\nroff2dvi\nroff2html\nroff2pdf\nroff2ps\nroff2text\nroff2x\nrpm\nrpm2archive\nrpm2cpio\nrpmargs\nrpmbuild\nrpmdb\nrpmdev-bumpspec\nrpmdev-checksig\nrpmdev-cksum\nrpmdev-diff\nrpmdev-extract\nrpmdev-md5\nrpmdev-packager\nrpmdev-rmdevelrpms\nrpmdev-setuptree\nrpmdev-sha1\nrpmdev-sha224\nrpmdev-sha256\nrpmdev-sha384\nrpmdev-sha512\nrpmdev-sort\nrpmdev-sum\nrpmdev-vercmp\nrpmdev-wipetree\nrpmdumpheader\nrpmfile\nrpmgraph\nrpmkeys\nrpmls\nrpmpeek\nrpmquery\nrpmsign\nrpmspec\nrpmverify\nRscript\nrsync\nrsync-ssl\nrunant.py\nruncon\nruntest\nrun-parts\nrview\nrvim\nsc\nscedit\nscinit\nscopenports\nscp\nscs2ascii\nscs2pdf\nscs2ps\nsc_install_defaults\nsdiff\nsed\nselfserv\nseq\nsetsid\nsftp\nsha1sum\nsha224sum\nsha256sum\nsha384sum\nsha512sum\nshasum\nshowrgb\nshow-changed-rco\nshow-installed\nshred\nshuf\nsigntool\nsignver\nsleep\nslencheck\nslsh\nsmtpd.py\nsoelim\nsort\nspectool\nsplain\nsplit\nsqlite3\nsqlite3_analyzer\nssh\nsshpass\nssh-add\nssh-agent\nssh-copy-id\nssh-keygen\nssh-keyscan\nssltap\nstat\nstatic-to-shared\nstdbuf\nstrsclnt\nstty\nsum\nsvr4-shrlib\nsxpm\nsymkeyutil\nsync\ntabs\ntac\ntail\ntar\ntbl\ntclsh8.6\ntdspool\ntee\ntest\ntexi2any\ntexi2dvi\ntexi2pdf\ntexindex\ntfmtodit\ntic\ntiff2bw\ntiff2pdf\ntiff2ps\ntiff2rgba\ntiffcmp\ntiffcp\ntiffcrop\ntiffdither\ntiffdump\ntiffinfo\ntiffmedian\ntiffset\ntiffsplit\ntimed-read\ntimed-run\ntimeout\ntjbench\ntmux\ntn5250\ntoe\ntouch\ntput\ntr\ntree\ntroff\ntrue\ntruncate\ntrust\ntset\ntsort\ntsql\ntstclnt\ntty\nuconv\nuname\nunbuffer\nuncompress\nunexpand\nuniq\nunix2dos\nunix2mac\nunix-lpr.sh\nunlink\nunlzma\nunpigz\nunxz\nunzip\nunzipsfx\nunzstd\nupdatedb\nupdate-alternatives\nupdate-ca-certificates\nuptime\nurlgrabber\nusers\nuuidgen\nuuidparse\nvacuumdb\nvacuumlo\nvba_extract.py\nvdir\nverifytree\nvfychain\nvfyserv\nview\nvim\nvimdiff\nvimtutor\nwatchgnupg\nwc\nwget\nwhatis\nwheel\nwhereis\nwho\nwhoami\nwish8.6\nwrjpgcom\nxargs\nxgettext\nxml2-config\nxmlcatalog\nxmllint\nxmlservice-cli\nxmlwf\nxsltproc\nxslt-config\nxsubpp\nxt5250\nxxd\nxz\nxzcat\nxzcmp\nxzdec\nxzdiff\nxzegrep\nxzfgrep\nxzgrep\nxzless\nxzmore\nyacc\nyat2m\nyes\nyum\nyumdownloader\nyum-builddep\nyum-config-manager\nyum-debug-dump\nyum-debug-restore\nyum-groups-manager\nzcat\nzcmp\nzdiff\nzegrep\nzfgrep\nzforce\nzgrep\nzip\nzipcloak\nzipcmp\nzipdetails\nzipgrep\nzipinfo\nzipmerge\nzipnote\nzipsplit\nziptool\nzless\nzmore\nznew\nzstd\nzstdcat\nzstdgrep\nzstdless\nzstdmt\n[\n__pycache__/",
    "stderr": ""
}

/DIRECTA1/home/paolos: ls -p /usr/bin/
{
    "code": null,
    "signal": null,
    "stdout": "ajar\nappletviewer\napt\narmsrv\narmsrvconv\nattr\nbasename\nbash\ncat\ncatsplf\nchgrp\nchmod\nchown\nclrtmp\ncmp\ncompress\ncp\ncut\ndataq\ndatarea\ndate\ndb2\ndb2profc\ndb2profc_trace\ndb2profp\nddns-confgen\ndelv\ndfmadmin\ndhcpd\ndhcrelay\ndig\ndirname\ndnssec-dsfromkey\ndnssec-importkey\ndnssec-keyfromlabel\ndnssec-keygen\ndnssec-revoke\ndnssec-settime\ndnssec-signzone\ndnssec-verify\ndspmsg\necho\negrep\nenv\nexpr\nextcheck\nfgrep\nfile\nfind\ngencat\ngetjobid\ngrep\nhead\nhost\nhostname\nhwkeytool\niconv\nid\nidlj\nikeyman\nipcrm\nipcs\nixlc\njar\njarsigner\njava\njavac\njavadoc\njavah\njavap\njconsole\njdb\njdmpview\njextract\nkdestroy\nkeytab\nkeytool\nkinit\nklist\nkpasswd\nksetup\nldapadd\nldapchangepwd\nldapcompare\nldapdelete\nldapdiff\nldapexop\nldapmodify\nldapmodrdn\nldapsearch\nln\nlocale\nlogger\nlogname\nls\nmkdir\nmkfifo\nmv\nnamed\nnamed-checkconf\nnamed-checkzone\nnamed-compilezone\nnamed-journalprint\nnamed-rrchecker\nnative2ascii\nnohup\nnslookup\nnsupdate\nod\norbd\npack200\npax\npolicytool\npr\nprintenv\nprofconv\nprofdb\nprofp\nps\npwd\npwdx\nqsh\nrexec\nrexx\nRfile\nrm\nrmdir\nrmic\nrmid\nrmiregistry\nrndc\nrndc-confgen\nschemagen\nsed\nserialver\nservertool\nsetccsid\nsh\nsort\nsplit\nsqlj\nstrEKM\nsystem\nsysval\ntail\ntar\ntee\ntest\ntnameserv\ntouch\ntr\ntsig-keygen\nuname\nuncompress\nuniq\nunpack200\nwc\nwsgen\nwsimport\nxargs\nxjc\nzcat",
    "stderr": ""
}

/DIRECTA1/home/paolos: ls -p /QSYS.lib/VSCODE.lib/
{
    "code": 2,
    "signal": null,
    "stdout": "",
    "stderr": "ls: cannot access '/QSYS.lib/VSCODE.lib/': Un file o una directory nel nome percorso non esiste."
}

/DIRECTA1/home/paolos: DB2UTIL_JSON_CONTAINER=array /QOpenSys/pkgs/bin/db2util -o json "SELECT * FROM QSYS2.ASP_INFO"
{
    "code": null,
    "signal": null,
    "stdout": "[\n{\"DEVICE_DESCRIPTION_NAME\":\"null\",\"ASP_NUMBER\":1,\"ASP_STATE\":\"NONE\",\"ASP_TYPE\":\"SYSTEM\",\"RDB_NAME\":\"DIRECTAC\",\"NUMBER_OF_DISK_UNITS\":3,\"DISK_UNITS_PRESENT\":\"ALL\",\"TOTAL_CAPACITY\":402802,\"TOTAL_CAPACITY_AVAILABLE\":214441,\"PROTECTED_CAPACITY\":402802,\"PROTECTED_CAPACITY_AVAILABLE\":214441,\"UNPROTECTED_CAPACITY\":0,\"UNPROTECTED_CAPACITY_AVAILABLE\":0,\"SYSTEM_STORAGE\":112,\"OVERFLOW_STORAGE\":0,\"STORAGE_THRESHOLD_PERCENTAGE\":90,\"OVERFLOW_RECOVERY_RESULT\":\"SUCCESS\",\"ERROR_LOG_SPACE\":1,\"MACHINE_LOG_SPACE\":1288,\"MACHINE_TRACE_SPACE\":1,\"MAIN_STORAGE_DUMP_SPACE\":1083,\"MICROCODE_SPACE\":5704,\"END_IMMEDIATE\":\"YES\",\"COMPRESSION_RECOVERY_POLICY\":\"OVERFLOW DELAY\",\"COMPRESSED_DISK_UNITS\":\"NONE\",\"CHANGES_WRITTEN_TO_DISK\":\"YES\",\"MULTIPLE_CONNECTION_DISK_UNITS\":\"NO\",\"BALANCE_STATUS\":\"COMPLETE\",\"BALANCE_TYPE\":\"NONE\",\"BALANCE_DATA_MOVED\":null,\"BALANCE_DATA_REMAINING\":null,\"BALANCE_TIMESTAMP\":\"null\",\"TRACE_STATUS\":\"NONE\",\"TRACE_DURATION\":null,\"TRACE_TIMESTAMP\":\"null\",\"RESOURCE_NAME\":\"null\",\"PRIMARY_ASP_RESOURCE_NAME\":\"null\"},\n{\"DEVICE_DESCRIPTION_NAME\":\"DIRECTA1\",\"ASP_NUMBER\":33,\"ASP_STATE\":\"AVAILABLE\",\"ASP_TYPE\":\"PRIMARY\",\"RDB_NAME\":\"DIRECTA1\",\"NUMBER_OF_DISK_UNITS\":7,\"DISK_UNITS_PRESENT\":\"ALL\",\"TOTAL_CAPACITY\":939872,\"TOTAL_CAPACITY_AVAILABLE\":743797,\"PROTECTED_CAPACITY\":939872,\"PROTECTED_CAPACITY_AVAILABLE\":743797,\"UNPROTECTED_CAPACITY\":0,\"UNPROTECTED_CAPACITY_AVAILABLE\":0,\"SYSTEM_STORAGE\":250,\"OVERFLOW_STORAGE\":null,\"STORAGE_THRESHOLD_PERCENTAGE\":90,\"OVERFLOW_RECOVERY_RESULT\":\"null\",\"ERROR_LOG_SPACE\":null,\"MACHINE_LOG_SPACE\":null,\"MACHINE_TRACE_SPACE\":null,\"MAIN_STORAGE_DUMP_SPACE\":null,\"MICROCODE_SPACE\":8,\"END_IMMEDIATE\":\"null\",\"COMPRESSION_RECOVERY_POLICY\":\"OVERFLOW IMMEDIATE\",\"COMPRESSED_DISK_UNITS\":\"NONE\",\"CHANGES_WRITTEN_TO_DISK\":\"YES\",\"MULTIPLE_CONNECTION_DISK_UNITS\":\"NO\",\"BALANCE_STATUS\":\"NONE\",\"BALANCE_TYPE\":\"NONE\",\"BALANCE_DATA_MOVED\":null,\"BALANCE_DATA_REMAINING\":null,\"BALANCE_TIMESTAMP\":\"null\",\"TRACE_STATUS\":\"NONE\",\"TRACE_DURATION\":null,\"TRACE_TIMESTAMP\":\"null\",\"RESOURCE_NAME\":\"DIRECTA1\",\"PRIMARY_ASP_RESOURCE_NAME\":\"null\"}\n]",
    "stderr": ""
}

**Comment by worksofliam** [comment]
@pabloto Thanks - my understanding is that I guess the extension continues to work okay for you, except you get this message appear every time you connect - which makes sense since we are not handling `CPF2158` at all.

The English translation is

```
CPF2158: Library X exists in ASP device ASP X.
```

Which means that it should be all good. I will need to add a little thing to handle this code.

Liam

**Comment by worksofliam** [comment]
Fixed in 0.8.22.
